packages/cli-kit/assets/cli-ruby/test/shopify-cli/theme/dev_server/proxy_test.rb

```diff
@@ -97,10 +97,10 @@ def test_new_webpixels_requests_are_ignored
 
         def test_get_is_proxied_to_theme_access_api_when_password_is_provided
           Environment.stubs(:theme_access_password?).returns(true)
-          Environment.stubs(:store).returns("https://dev-theme-server-store.myshopify.com")
+          Environment.stubs(:store).returns("dev-theme-server-store.myshopify.com")
```

Comment:
With `https://`, `uri = URI.join("https://#{shop}", path)` is behaving weird (`https://https…`).
It didn't matter for this test, but for the added one.